### ISSUE TITLE
Two squads going to different points may not engage in combat.

### ISSUE THREAD

**Issue by SurDno** [Decane/SRP#31]
![image](https://user-images.githubusercontent.com/35204405/47448844-bcc18e80-d7c9-11e8-8311-78a7b0ea95df.png)
As you can see, these two squads of bandits and loners got past each other and did not engage in combat. I have a save reproducing it: https://yadi.sk/d/kS8evOGTUDvkfw

**Comment by Decane** [comment]
Yes, I have witnessed this as well. The logic in xr_combat_ignore.script is the likely culprit. By the way, you uploaded just the thumbnail (.dds) of your savegame, not the save file (.sav) itself.

**Comment by SurDno** [comment]
Oh, sorry, I've been always mixing these up. :В
Here's the actual savegame: https://yadi.sk/d/eJDkLNGW12gVZw
